content/en/account_management/saml/_index.md

```diff
@@ -20,9 +20,33 @@ Configuring [SAML (Security Assertion Markup Language)][1] for your Datadog acco
 
 **Notes**: 
 
-- If you don't have SAML enabled on your Datadog account, reach out to [support][2] to enable it.
-- This documentation assumes that you already have a SAML Identity Provider (IdP). If you do not have a SAML IdP, there are several IdPs that have integrations with Datadog such as [Active Directory][3], [Auth0][4], [Azure][3], [Google][5], [LastPass][6], [Okta][7], and [SafeNet][8].
-- SAML configuration requires [Datadog Administrator][9] access.
+{{% site-region region="us,us3,us5,eu,ap1" %}}
+- If you don't have SAML enabled on your Datadog account, reach out to [support][1] to enable it.
+- This documentation assumes that you already have a SAML Identity Provider (IdP). If you do not have a SAML IdP, there are several IdPs that have integrations with Datadog such as [Active Directory][2], [Auth0][3], [Azure][2], [Google][4], [LastPass][5], [Okta][6], and [SafeNet][7].
+- SAML configuration requires [Datadog Administrator][8] access.
+
+[1]: /help/
+[2]: https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/auth-saml
+[3]: https://auth0.com/docs/protocols/saml-protocol
+[4]: https://cloud.google.com/architecture/identity/single-sign-on
+[5]: https://support.logmeininc.com/lastpass/help/lastpass-admin-toolkit-using-single-sign-on-sso
+[6]: https://developer.okta.com/docs/concepts/saml/
+[7]: https://help.safenetid.com/operator/Content/STA/Apps/Apps_SAML.htm
```

Comment:
This link appears to be broken. I did a search and it might live here now: https://thalesdocs.com/sta/operator/applications/apps_saml/index.html

Reply:
You're right. Thank you!